pylint.rc

```diff
@@ -35,7 +35,8 @@ load-plugins=
 # it should appear only once).
 #    R0201: *Method could be a function*
 #    C0111: *Missing doc string*
-disable=E1101,R0201,W0212,C0111
+#    R0903: *Too few public methods*
```

Comment:
Ha!

You can also put this in the top of jitcheck.py.  I don't really have a preference.